pandas/tests/reshape/merge/test_multi.py

```diff
@@ -481,6 +481,54 @@ def test_merge_datetime_index(self, klass):
         result = df.merge(df, on=[df.index.year], how="inner")
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("merge_type", ["left", "right"])
+    def test_merge_datetime_multi_index_empty_df(self, merge_type):
+        # see gh-36895
+
+        midx1 = MultiIndex.from_tuples(
+            [[Timestamp("1950-01-01"), "A"], [Timestamp("1950-01-02"), "B"]],
+            names=["date", "panel"],
+        )
+        left = DataFrame(
+            data={
+                "data": [1.5, 1.5],
+            },
+            index=midx1,
```

Comment:
can you put the index directly in-line with the frame constructions, these midx1 and midx2 are just not meaningful names